package.json

```diff
@@ -3,15 +3,14 @@
   "version": "2.0.0",
   "description": "Connect to web data from Tableau.",
   "scripts": {
-    "corsproxy": "export CORSPROXY_PORT=8889 || set CORSPROXY_PORT=8889 && node ./Simulator/utils/corsproxy.js",
```

Comment:
This is the migrated cors-proxy that could be run with npm start already, did you accidentally remove this?

Reply:
Yes, I was initially looking at another PR that avoided using `corsproxy`, but I reverted this change for inclusion :)